### ISSUE TITLE
[Enhancement]: A way to see which audiobooks have missing ebooks

### ISSUE THREAD

**Issue by danmed** [advplyr/audiobookshelf#2665]
### Describe the feature/enhancement

I'm on a mission to make sure i have an ebook for every audiobook in my library but i took a good 6 month break in my mission and now i'm trying to find a way to easily see which ebooks i'm missing..

Is there any where i can see this or a way i can query the database to figure it out?

**Comment by nichwall** [comment]
There is not a way to do it within the interface (duplicate of https://github.com/advplyr/audiobookshelf/issues/1889) but you could look at the database in the Books table to filter by which entries have ebooks.

**Comment by advplyr** [comment]
Yeah you can query the sqlite db for that. `select * from books where ebookFile is null`

**Comment by noahbaculi** [comment]
Hey @danmed, I know it's been a while but I have been on the same mission of matching an ebook to every audiobook! The [ABS filter](https://github.com/advplyr/audiobookshelf/issues/1889) is nice but the copy pasting still gets annoying with the sheer number of audiobooks I have 😅 I ended up building a small self-hosted app to facilitate my workflow:
https://github.com/noahbaculi/missing-ebooks

A couple friends and I have been running it for a bit and it's at a point where I'm confident opening it up to others now. It runs alongside Audiobookshelf rather than inside it, so the audiobooks don't actually have to be in ABS. This tool just presents a tree of every folder that holds audio with no ebook covering it. Marking a folder writes a `.no_ebook` or `.ebook_elsewhere` file into the library, so the state lives on disk and there's no database to back up. Audio and ebook extension lists follow the Audiobookshelf scanner defaults

There's a [live demo with dummy data](https://demo-missing-ebooks.noahbaculi.com) if you want to try it out before pointing it at your library! I'd love any feedback - I know everyone's library and organization is different
